tasks/main.yml

```diff
@@ -420,6 +420,7 @@
     - name: Copy certificate and private_key files to the host
       copy:
         src: "{{ item }}"
+        remote_src: "{{ mssql_tls_remote_src }}"
```

Comment:
So this assumes the cert and key are already on the remote host, just not in the right location?

Reply:
Yes, e.g. when using the certificate role to generate certificates and then must move them to the location required for mssql